casa/OS/HostInfo.h

```diff
@@ -131,12 +131,24 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Allows to set custom resource values overriding the
+    // existing .apisrc setting and/or system defaults
+    // Returns the value previously stored
+    // <group>
+    static ptrdiff_t setMemoryTotal(ptrdiff_t memory);
```

Comment:
missing documentation on the expected units which should be KiB for memtotal and % for memfrac